app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="card text-bg-dark mb-4" id="<%= dom_id recipe %>">
+  <div class="card-body">
+    <h3 class="card-title"><%= recipe.name %></h3>
```

Comment:
- [ ] Please make sure that clicking on a recipe item takes you to the recipe details page `/recipes/:id/` as shown in the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true) :pray: